packages/rari/src/runtime/rsc-client-runtime.ts

```
@@ -1,6 +1,55 @@
+  '~rscRefreshCounters'?: Record<string, number>
+}
+
+interface WindowWithRari extends Window {
+  '~rari': GlobalWithRari['~rari']
+  '~clientComponents': GlobalWithRari['~clientComponents']
+  '~clientComponentPaths': GlobalWithRari['~clientComponentPaths']
+  '~clientComponentNames': GlobalWithRari['~clientComponentNames']
+  '~rscRefreshCounters'?: Record<string, number>
+}
+
+declare let globalThis: GlobalWithRari
```

Biome (2.3.13)

[error] Do not shadow the global "globalThis" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)